pkgs/applications/misc/tango/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "tango";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "masakichi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-e/M2iRm/UwfnRVnMo1PmQTkz4IGTxnsCXNSSUkhsiHk=";
+  };
+
+  vendorSha256 = "sha256-83nKtiEy1na1HgAQcbTEfl+0vGg6BkCLBK1REN9fP+k=";
```

Comment:
Can we use `vendorHash` instead of `vendorSha256` in `buildGoModule` ?

Reply:
Fixed